openapi.yaml

```diff
@@ -12257,11 +13280,11 @@ paths:
                         from: '2024-05-28T11:08:39+00:00'
                         to: '2024-05-29T11:08:39+00:00'
                     data:
-                    - emittedAt: '2024-05-29T07:00:00+00:00'
+                    - emittedAt: '2024-05-29T07+00:00:00:00'
                       metricValue: 2
-                    - emittedAt: '2024-05-29T08:00:00+00:00'
+                    - emittedAt: '2024-05-29T08+00:00:00:00'
                       metricValue: 1
-                    - emittedAt: '2024-05-29T09:00:00+00:00'
+                    - emittedAt: '2024-05-29T09+00:00:00:00'
```

Comment:
Not sure this is valid ?

Reply:
Hmm I think you are right, this one is not correct  😬 I'll fix this.